gitlabform/processors/project/hooks_processor.py

```diff
@@ -9,23 +11,26 @@ def __init__(self, gitlab: GitLab):
         super().__init__("hooks", gitlab)
 
     def _process_configuration(self, project_and_group: str, configuration: dict):
+        debug("Processing hooks...")
+        project: Project = self.gl.projects.get(project_and_group)
+        hooks_list = project.hooks.list()
```

Comment:
For consistency, let's apply types to all the variables (i.e. `hooks_list`, `hook`, `hook_id`, `attr`, etc.)